server/src/main/java/org/candlepin/resource/dto/HypervisorUpdateResult.java

```diff
@@ -46,6 +46,12 @@ public HypervisorUpdateResult() {
         this.failed = new HashSet<String>();
     }
 
+    public boolean wasCreated(Consumer c) { return created.contains(c); }
+
+    public boolean wasUpdated(Consumer c) { return updated.contains(c); }
+
+    public boolean wasUnchanged(Consumer c) { return unchanged.contains(c); }
```

Comment:
Does this make it through the linter? We don't have any one-liners like this anywhere else; should use normal formatting here.